python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/port-channel-interfaces.j2

```diff
@@ -723,6 +723,11 @@ interface {{ port_channel_interface.name }}
    link tracking group {{ link_tracking_group.name }} {{ link_tracking_group.direction }}
 {%         endif %}
 {%     endfor %}
+{%     if port_channel_interface.link_tracking.direction is arista.avd.defined %}
+{%         for group_name in port_channel_interface.link_tracking.groups | arista.avd.natural_sort %}
+   link tracking group {{ group_name }} {{ port_channel_interface.link_tracking.direction }}
```

Comment:
i dont think we need sorting here.
```
wan2-dc1(config-if-Po1)#sh ac
interface Port-Channel1
   link tracking group mahesh upstream
   link tracking group kumar upstream
```

Reply:
You are right!
I checked it like:

```
link tracking group aa upstream
link tracking group kk upstream
link tracking group vibhu upstream
```

but now I see its random:
```
   link tracking group cc upstream
   link tracking group aa upstream
   link tracking group mm upstream
   link tracking group kk upstream
   link tracking group mahesh upstream
   link tracking group bb upstream
   link tracking group kumar upstream
   link tracking group vibhu upstream
```

Reply:
Fixed!